docs/HACKING.md

```diff
@@ -7,7 +7,8 @@
 - The operator is written using operator-sdk 
 [v1.6.1](https://github.com/operator-framework/operator-sdk/releases/tag/v1.6.1) and has the same 
 [pre-requisites](https://sdk.operatorframework.io/docs/installation/#prerequisites) as it does.
-
+- Before running build command make sure to refer `Updating Git submodules` section in this document and follow the steps.
```

Comment:
`Updating Git submodules` should link to the section.
Additionally this should be framed as a general pre-req, it being a pre-req implies it is done before running the build commands:
```
-  git submodules are initialized and up to date
```